src/page-multi-facility/FacilityRowPlaceholder.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useRef, useState } from "react";
+import useVisibility from "react-use-visibility";
+
+import Loading from "../design-system/Loading";
+import { Facility } from "./types";
+
+interface Props {
+  facility: Facility;
```

Comment:
it doesn't seem like you are actually using this prop?

Reply:
Oh good point. I originally was, then I reverted the change.